improvelib/initializer/config.py

```diff
@@ -69,52 +71,32 @@ def load_config(self):
             self.logger.error("Can't load config from %s", str(self.file))
             self.config['DEFAULT'] = {}
 
-    def save_config(self, file, config=None):
-        """ TODO ... """
-        if os.path.isabs(file):
-            self.config.write(file)
+    def save_parameter_file(self, file_name):
```

Comment:
The config object does not have the parameters that are used. Agree that INI (and ideally a choice of multiple formats would be ideal), this will go on the to do list but not for today.